src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalForV2ApiTest.java

```diff
@@ -204,4 +212,324 @@ public void findOrganisationPbaWithoutEmailByExternalUserShouldBeBadRequestV2()
+                "dxUpdatedNum","dxUpdatedExchange");
+
+        //call endpoint to update contactInformation
+        Response orgUpdatedResponse = professionalApiClient.updateContactInformationDetails(
+            updateContactInformationRequest,professionalApiClient.getMultipleAuthHeaders(pomBearerToken));
+        Assertions.assertNotNull(orgUpdatedResponse);
+        assertThat(orgUpdatedResponse.getBody()
+            .prettyPrint()).contains("AddressLine1 cannot be empty");
+        assertThat(orgUpdatedResponse.statusCode()).isEqualTo(400);
+        log.info("updateContactInformationFailureAddressLine1Missing :: END");
+
+    }
+
+
+    @Test
+    void updateContactInfoFailureDxNumberEmpty() {
+        setUpOrgTestData();
+        setUpUserBearerTokens(List.of(puiOrgManager));
+        log.info("updateContactInfoFailureDxNumberEmpty :: STARTED");
+
+        UpdateContactInformationRequest updateContactInformationRequest =
+            new UpdateContactInformationRequest("UPRN1234567891234567898",
+                "updatedaddressLine1","updatedaddressLine2","updatedaddressLine3",
+                "updatedtownCity","updatedcounty","updatedcountry","updatedpostCode",
+                "","dxUpdatedExchange");
+
+        //call endpoint to update contactInformation
+        Response orgUpdatedResponse = professionalApiClient.updateContactInformationDetails(
+            updateContactInformationRequest,professionalApiClient.getMultipleAuthHeaders(pomBearerToken));
+        Assertions.assertNotNull(orgUpdatedResponse);
+        assertThat(orgUpdatedResponse.getBody()
+            .prettyPrint()).contains("Uprn must not be greater than 14 characters long found: "
+            + "UPRN1234567891234567898".length());
+        assertThat(orgUpdatedResponse.statusCode()).isEqualTo(400);
+        log.info("updateContactInfoFailureDxNumberEmpty :: END");
+
+    }
+
+    @Test
+    void updateContactInfoAndDxAddressWhenUprnLength15Failure() {
+        setUpOrgTestData();
+        setUpUserBearerTokens(List.of(puiOrgManager));
+        log.info("updateContactInfoAndDxAddressWhenUprnLength15Failure :: STARTED");
+
+        UpdateContactInformationRequest updateContactInformationRequest =
+            new UpdateContactInformationRequest("UPRN1234567891234567898",
+                "updatedaddressLine1","updatedaddressLine2","updatedaddressLine3",
+                "updatedtownCity","updatedcounty","updatedcountry","updatedpostCode",
+                "dxNumUpdated","");
+
+        //call endpoint to update contactInformation
+        Response orgUpdatedResponse = professionalApiClient.updateContactInformationDetails(
+            updateContactInformationRequest,professionalApiClient.getMultipleAuthHeaders(pomBearerToken));
+        Assertions.assertNotNull(orgUpdatedResponse);
+        assertThat(orgUpdatedResponse.getBody()
+            .prettyPrint()).contains("Uprn must not be greater than 14 characters long found: 23");
```

Comment:
Wrong error message, update UPRN to correct length and verify for correct error message.